internal/delete.go

```diff
@@ -40,7 +40,7 @@ func checkIfAppFolderExists(pathToDelete string) {
 		} else {
 			fmt.Println("Error checking app folder: ", err)
 		}
-		os.Exit(1)
+		panic(err)
```

Comment:
```go
if os.IsNotExist(err) {
	panic("App folder not found")
} else {
	panic("Error checking app folder: "+ err.Error())
}
```

Reply:
A mensagem 'App folder not found' não é um erro. É melhor evitar o uso excessivo do `panic`, pois isso dá a impressão de que a ferramenta (**Boneless CLI**) está quebrando, já que exibe várias informações de stacktrace com números de linha.